From 76a206f8b049aa41e234a489b673ac3431db8b43 Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Mon, 23 Oct 2023 10:51:12 -0700
Subject: [PATCH 09/12] Add Clear Linux cert_path

Change '/etc/pki/ca-trust' to the correct CL path.

Changes for the mock 3.0 release: Prepend the CL path to the new
`cert_paths` list instead.

Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
Signed-off-by: William Douglas <william.douglas@intel.com>
---
 py/mockbuild/package_manager.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/py/mockbuild/package_manager.py b/py/mockbuild/package_manager.py
index 6f0b4e7..38e5273 100644
--- a/py/mockbuild/package_manager.py
+++ b/py/mockbuild/package_manager.py
@@ -397,7 +397,7 @@ Error:      Neither dnf-utils nor yum-utils are installed. Dnf-utils or yum-util
 
     @traceLog()
     def copy_certs(self):
-        cert_paths = ["/etc/pki/ca-trust", "/usr/share/pki/ca-trust-source"]
+        cert_paths = ["/var/cache/ca-certs/anchors", "/etc/pki/ca-trust", "/usr/share/pki/ca-trust-source"]
         for cert_path in cert_paths:
             pki_dir = self.buildroot.make_chroot_path(cert_path)
             file_util.update_tree(pki_dir, cert_path)
-- 
2.42.0

